app/players/elves/druid.py

```diff
@@ -0,0 +1,21 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            favourite_spell: str
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        return (
+            f"Druid {self.nickname}. "
+            f"{self.nickname} has a favourite spell: {self._favourite_spell}"

```

Comment:
Consider using double quotes for consistency, as per the checklist requirement to use one style of quotes in your code. Double quotes are preferable.